### ISSUE TITLE
[SF][ui5-form-group]:Add accessibleName prop to the FormGroup

### ISSUE THREAD

**Issue by VidhiNPatel** [UI5/webcomponents#12377]
### Feature Request Description

Created the feature request after this discussion topic - https://github.com/UI5/webcomponents/discussions/12159#discussioncomment-14442514

Bug Description
Currently, Axe Devtool is reporting a missing aria-label on the where role="form" is specified. According to accessibility standards, an aria-label should be present when a role is defined.

<img width="3396" height="1488" alt="Image" src="https://github.com/user-attachments/assets/5d3a3d10-9747-47c0-8f33-1dbc1549de00" />

<img width="2716" height="1490" alt="Image" src="https://github.com/user-attachments/assets/a2d71a56-7b9d-4105-80cf-a7d897a42b94" />

### Proposed Solution

`ui5-form-group` component should have accessibleName prop

### Priority

Medium

### Privacy Policy

- [x] I’m not disclosing any internal or sensitive information.

**Comment by dobrinyonkov** [comment]
Hi @UI5/ui5-team-webc, could you please take over this feature request?

Best Regards,
Dobrin
